documentation/docs/30-advanced/20-hooks.md

```diff
@@ -187,6 +188,8 @@ const Sentry: any;
 
 // @filename: index.js
 // ---cut---
+import * as Sentry from '@sentry/svelte';
+
 /** @type {import('@sveltejs/kit').HandleClientError} */
 export function handleError({ error, event }) {
```

Comment:
In both files (client and server), we also have to initialize the Sentry SDK so that errors are caught. 

(btw, take this exact suggestion with a grain of salt, I'll leave it up to you how detailed you want to document the `Sentry.init` call)

```suggestion
import * as Sentry from '@sentry/svelte';

Sentry.init({...})

/** @type {import('@sveltejs/kit').HandleClientError} */
export function handleError({ error, event }) {
```